lighthouse/types.py

```diff
@@ -12,28 +12,40 @@
 EventMessage = Dict[str, Union[str, Event]]
 
 
+class EventPropertyProtocol(Protocol):
+    @property
+    def _params(self) -> Dict[str, str]:
+        ...
+
+    def process_validation(self, condition: bool, message: str) -> None:
+        ...
+
+    def validation_scope(self):
+        ...
+
+    def is_integer(self, n: Optional[str]) -> bool:
+        ...
+
+
 class PlateEvent(Protocol):
     """This class is used to assist the mixin while type checking."""
 
     @property
-    def plate_barcode(self) -> str:
+    def properties(self) -> Dict[str, Any]:
         ...
 
     @property
-    def robot_serial_number(self) -> str:
+    def plate_barcode(self) -> str:
         ...
 
-
-class EventPropertyProtocol(Protocol):
     @property
-    def _params(self) -> Dict[str, str]:
+    def robot_serial_number(self) -> str:
         ...
 
-    def process_validation(self, condition: bool, message: str) -> None:
-        ...
 
-    def validation_scope(self):
-        ...
+class SourceNoPickableSamples(PlateEvent):
```

Comment:
Hmm I'm not too sure of these empty classes and they can also be different dependent on the system.

Reply:
Not needed anymore, I'll remove

Reply:
Done